.eslintignore

```diff
@@ -0,0 +1,7 @@
+/build/**
+/api_docs/**
+**/node_modules/**
+**/test/**/*
```

Comment:
Shall we take test files into account as well?

Reply:
Enabling it for test adds additional 101 errors.. we can create a ticket later to address those and remove test from ignore list.. Right now I have kept it how it was with tslint.json were test files were ignored..

Reply:
Gotcha, that makes sense. Would you mind creating a ticket or a reminder somewhere so that we could revisit it later?

Reply:
done